internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+type PullRequestEvent struct {
+	Action      string      `json:"action"`
```

Comment:
Consider adding validation for the struct members, especially for IDs and URLs to ensure they conform to expected formats. This will help avoid logical errors when processing incoming events.